common/src/main/java/zm/gov/moh/common/submodule/form/view/FormFragment.java

```diff
@@ -88,6 +89,25 @@ public View onCreateView (LayoutInflater inflater, ViewGroup container,
             Exception ex = e;
         }
 
+        //Criteria,Action
+        if( formModel.getAttributes().getLogic() != null)
+            for(Logic logic : formModel.getAttributes().getLogic()){
+
+                if(logic.getAction().getType().equals(Action.ACTION_TYPE_CRITERIA)){
+
+                    for(String tag : logic.getAction().getMetadata().getTags())
+                        if(bundle.containsKey(tag)) {
+
+                            String value = bundle.getString(tag);
+                            if(!value.equals(logic.getCondition().getValue())){
+                                context.onBackPressed();
+                                Toast.makeText(context,"Male Patients cannnot enroll for this service",Toast.LENGTH_SHORT).show();
```

Comment:
@csamiselo 
1. Correct typo `cannnot` 
3. Lower case letter `p` on `Patients`
3. Move String literal `"Male Patients cannnot enroll for this service"` to XML resources incase a requirement for multi language support arises.

Reply:
Alright will make amendments